bin/i18n/i18n_metrics.rb

```diff
@@ -0,0 +1,37 @@
+require 'cdo/aws/metrics'
+require 'aws-sdk-ec2'
+require 'net/http'
+require 'socket'
+
+module I18n
+  module Metrics
+    I18N_METRICS_NAMESPACE = 'I18n'.freeze
+
+    def self.report_runtime(addtl_dimensions)
+      log_metric(:Runtime, addtl_dimensions, yield)
+    end
+
+    def self.machine_id
+      @machine_id ||= begin
+        Net::HTTP.get(URI.parse(CDO.ec2_metadata_endpoint + 'instance-id'))
+      rescue
+        'local_machine'
+      end
+    end
+
+    def self.log_metric(metric_name, addtl_dimensions, value)
```

Comment:
It would be nice to change the sequence of the arguments by their priority: `def self.log_metric(metric_name, value, addtl_dimensions)`